src/transforms/CSSHelper.ts

```diff
@@ -283,7 +297,7 @@ export class CSSHelper {
     // Formal CSS definition: font-style, font-variant, font-weight, font-stretch, font-size, line-height, and font-family.
     // Example: small-caps bold 24px/1rem "Wingdings"
     const fragmentCaps = data.caps ? 'small-caps ' : ''
-    const fragmentWeight = fontWeightReference ? data.fontWeight : `\"${data.fontWeight}\"`
+    const fragmentWeight = fontWeightReference ? data.fontWeight : `${data.fontWeight}`
```

Comment:
What does the condition do actually? It seems it will always return the same thing, or doesn't? 🤔 

In other words, `data.fontWeight` is the same thing as `${data-fontWeight}` afaik

Reply:
So it could effectively be:
```
const fragmentWeight = data.fontWeight
```